cmd/supermarketSearch.go

```diff
@@ -16,7 +16,7 @@ var query, format string
 
 // supermarketSearchCmd represents the supermarket search
 var supermarketSearchCmd = &cobra.Command{
-	Use:   "search",
+	Use:   "search <artifact-type> <artifact-name>",
```

Comment:
here, it would be good to specify "search <artifact-type> <keyword>"

Reply:
Done.